src/pages/user/Recipes.js

```diff
@@ -9,8 +9,10 @@ export default class UserRecipes extends Component {
 
     for (let i = 0; i < 20; i++) {
       listItems.push(
-        <tr>
-          <Link to="#">Recipe {i}</Link>
+        <tr key={i}>
```

Comment:
I rolled this fix into #13, so GH should figure it out but you might need to rebase against master if it doesn't want to auto-merge for you.